python/ray/_private/services.py

```diff
@@ -122,6 +122,40 @@ def new_port():
 
 
 def find_redis_address(address=None):
+    """
+    Currently, this extracts the --redis-address from the command that launched
```

Comment:
Let's call this `--address` instead of `--redis-address` everywhere in the comment (`--redis-address` is the deprecated name).

Reply:
I've added a note that `--redis-address` is deprecated, but I think that the fact that this function looks for `--redis-address` instead of `--address` is literally the most important thing to know about this function, precisely because it's using the deprecated name.

Right now, if something is going wrong and someone comes poking through this code, that `--redis-address` sure jumps out, and it's helpful to have something clarifying "If you check ps aux and the command line looks like this, then the problem is not here."

And in the future, when the relevant command lines actually are changed to remove `--redis-address` --- I mean, this function is going to inevitably break, right? Because in the actual code it is in fact looking for `--redis-address`. So having a warning here might save some time for the poor developer trying to remove `--redis-address`.

Reply:
I didn't actually realize that we hadn't changed `--redis-address` internally! This makes sense to me, thanks.

Reply:
Yeah, it sounds like he didn't realize either until he started digging into it: https://github.com/ray-project/ray/pull/11735
I don't expect the docstring to help *much*, but hopefully it'll help at least a little the next time something like https://github.com/ray-project/ray/pull/11735 comes up.